fsharp-backend/tests/FuzzTests/Generators.fs

```diff
@@ -25,7 +25,7 @@ module DvalReprInternal = LibExecution.DvalReprInternal
 let isSafeOCamlString (s : string) : bool = s <> null && not (s.Contains('\u0000'))
 
 /// We disallow `\u0000` in OCaml because Postgres doesn't like it; see `of_utf8_encoded_string.ml`
-/// FSTODO: add in unicode
+/// FSTODO: add in unicode WHATISTHIS
```

Comment:
What's the expectation here?

Reply:
To generate all unicode (except \u0000). If we already do that here then it's done.